app/assets/json/translations/locale-en.json

```diff
@@ -565,6 +566,7 @@
         "PLEASE_SELECT_ONE": "Please select one.",
         "PLEASE_SELECT_PREFERENCE": "If you have one of these lottery preferences, select it below:",
         "PREAPPROVAL_LETTER": "Pre-approval letter",
+        "PREAPPROVAL_LETTER_ATTACHED": "Pre-approval letter attached",
```

Comment:
could probably use the file_attached if I can put translated "Pre-approval" in there too.

Reply:
ended up moving this to the component